backend-api/src/functions/common/session/updateOperations/BiometricSessionFinished.ts

```diff
@@ -0,0 +1,23 @@
+import { IUpdateSessionOperation, SessionState } from "../Session";
+
+export class BiometricSessionFinished implements IUpdateSessionOperation {
+  constructor(private readonly biometricSessionId: string) {}
+
+  getDynamoDbUpdateExpression() {
+    return "set biometricSessionId = :biometricSessionId, sessionState = :biometricSessionFinished";
+  }
+
+  getDynamoDbConditionExpression(): string {
+    return `sessionState in (:biometricTokenIssued)`;
+  }
+
+  getDynamoDbExpressionAttributeValues() {
+    return {
```

Comment:
Not relevant to this POC but for future ref, there are marshall and unmarshall functions that deal with that silly { S: 'something' } formatting. Believe @anahickmann may have used them in the async codebase already so we could look to use them in future.

Reply:
Yeah I realised that afterwards - let's use those for the real thing